cmd/influx/cli/cli.go

```diff
@@ -746,6 +796,30 @@ func (c *CommandLine) ExecuteQuery(query string) error {
 	return nil
 }
 
+var errQueryAborted = errors.New("query aborted")
+
+// clientDoQuery executes the query and calls f with the response. If a Signal is sent
+// or ctx.Done is closed while the query of f is running, it cancels the query
+// waits for f to exit, and returns an errot. Otherwise it returns f's error.
+func clientDoQuery(ctx context.Context, cancel context.CancelFunc, c *CommandLine, query string, f func(*client.Response, error) error) error {
+	// Run the HTTP request in a goroutine and pass the response to f.
+	ch := make(chan error, 1)
+	go func() { ch <- f(c.Client.QueryWithContext(ctx, c.query(query))) }()
+	select {
+	case <-c.osSignals:
+		cancel()
+		fmt.Println("Aborting query..")
```

Comment:
I don't think we need to print that the query was aborted. The person knows what they did.

Reply:
Done. I took the message out, but on my opinion when testing this on the cli I found returning a "Cancelled" message to be clearer. That's my opinion as a User.